docs/infrastructure.md

```
@@ -0,0 +1,228 @@
+  group:
+    blue: [prod, blue]
+    green: [prod, green]
+```
+
+---
+
+## CI/CD 파이프라인
+
+### develop 브랜치 (dev 배포)
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)